docker/storage/cassandra/run.sh

```diff
@@ -1,7 +1,7 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
I verified this manually also, there's no bash things we use here even when doing things like IP lookup